src/objects.c

```diff
@@ -387,6 +387,36 @@ PgUser *add_user(const char *name, const char *passwd)
 	return user;
 }
 
+/* find original or preconfigured user given the login user */
+PgUser *find_original_user(PgUser *login_user)
```

Comment:
This function name isn't really descriptive of what this function is doing, especially the `put_in_order` call.

Reply:
`find_original_user` no longer exists now.